src/components/CommitsPage/CommitGraphWrapper.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { ApolloError } from "@apollo/client";
+import styled from "@emotion/styled";
+import { Array } from "@ungap/global-this";
+import { Skeleton } from "antd";
+import { PageWrapper } from "components/styles";
+import { MainlineCommitsQuery } from "gql/generated/types";
+import { CommitGraph } from "./CommitGraph";
+
+export const CommitGraphWrapper: React.FC<{
+  versions: MainlineCommitsQuery["mainlineCommits"]["versions"];
+  error?: ApolloError;
+  graphType: "percentage" | "absolute";
+}> = ({ versions, error, graphType }) => {
+  let max = -1;
+  if (error) {
+    return <PageWrapper>ERROR</PageWrapper>;
+  }
+  if (!versions) {
+    return <StyledSkeleton active title={false} paragraph={{ rows: 4 }} />;
+  }
+  if (graphType === "absolute") {
+    max = findMax(data);
+  }
+  if (versions?.length !== 0) {
+    return (
+      <>
+        <FlexRowContainer>
+          {data.map((value) => (
+            <CommitGraph taskCounts={value} max={max} graphType={graphType} />
```

Comment:
you can replace `max={max}` with `max={graphType === "absolute" ? findMax(data) : -1}`